flow/scripts/global_place.tcl

```diff
@@ -30,8 +30,13 @@ if {[info exist ::env(PLACE_DENSITY_LB_ADDON)]} {
 
 set global_placement_args {}
 if {$::env(GPL_ROUTABILITY_DRIVEN)} {
-  lappend global_placement_args {-routability_driven}
+    lappend global_placement_args {-routability_driven}
+    if { [info exists ::env(GPL_TARGET_RC)] } { 
+	lappend global_placement_args {-routability_target_rc_metric}
```

Comment:
nit: lappend takes multiple arguments, so you can have a single line here. Remove the `{` escaping around arguments, not needed/idiomatic in Tcl.